src/Contrib/Zipkin/SpanKind.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Contrib\Zipkin;
+
+final class SpanKind
+{
+    public const KIND_SERVER = 'SERVER';
+    public const KIND_CLIENT = 'CLIENT';
+    public const KIND_PRODUCER = 'PRODUCER';
+    public const KIND_CONSUMER = 'CONSUMER';
+
+    public static function getChoices(): array
```

Comment:
this looks like it is not used? and the following constructor is redundant too

Reply:
@brettmc 
This is following the current implementation of [API/Trace/SpanKind](https://github.com/open-telemetry/opentelemetry-php/blob/01c388e02cd0dd94915475d4b4032bbd8aaa8e93/src/API/Trace/SpanKind.php)

However the choices there could be a constant as well:
```php
public const CHOICES = [
            self::KIND_CLIENT,
            self::KIND_SERVER,
            self::KIND_PRODUCER,
            self::KIND_CONSUMER,
        ];
        
```
While I don't know if if "choices" is the best wording here. These are more like roles imho.

Reply:
Right, that makes sense. I don't see any evidence of them being used (and it is not covered by any tests) - but I could see it being potentially useful for SDK users etc.

Reply:
Yeah, I've tried making similar implementation of [API/Trace/SpanKind](https://github.com/open-telemetry/opentelemetry-php/blob/01c388e02cd0dd94915475d4b4032bbd8aaa8e93/src/API/Trace/SpanKind.php) but I'm not too sure that was necessary. Currently there's no real usage of `getChoices` so I could simply leave this class with constants only. Potentially it could become a pure enum in latter versions of php.

Oh and another thought about these constants - having `KIND_` prefix on them is a bit redundant since constant purpose is already implied with class name. Maybe I should remove it to have usages like `SpanKind::CLIENT` instead of `SpanKind::KIND_CLIENT`?

Reply:
> Oh and another thought about these constants - having KIND_ prefix on them is a bit redundant since constant purpose is already implied with class name. Maybe I should remove it to have usages like SpanKind::CLIENT instead of SpanKind::KIND_CLIENT?

I have noticed that, too. Just go ahead and remove the "SPAN_" prefixes. I Think it looks cleaner that way.

Reply:
I've renamed SpanKind constants and removed the `getChoices` method. Let me know if there's any additional changes you want to be done :)

Reply:
Look good to me. Thanks for your contribution!

>  Let me know if there's any additional changes you want to be done :)

You could change the original [API/Trace/SpanKind](https://github.com/open-telemetry/opentelemetry-php/blob/01c388e02cd0dd94915475d4b4032bbd8aaa8e93/src/API/Trace/SpanKind.php) accordingly, if you want, but that would probably be better done in another PR.